golangci.yml

```diff
@@ -73,6 +73,7 @@ linters:
     - musttag
```

Comment:
Commenting a random line as GitHub mobile app sucks (bugs)

I would like to recommend you configuring gci

You enable it with enable-all, but it requires some configuration

Here is what I would like to suggest you

https://github.com/ccoVeille/golangci-lint-config-examples/blob/main/03-safe%2F.golangci.yml#L53-L60

Reply:
does it do more stringent than gofumpt which I have on? usually that's what triggers for me

also... isn't the default config of gci more specific than what you linked?

https://golangci-lint.run/usage/linters/#gci

Reply:
edit default is     # Default: ["standard", "default"]

Reply:
Exactly

Reply:
It won't conflict with gofumpt.

Moving the local module out of the default one allows to have clear ideas of what is imported from projects

But for fortio, maybe `Prefix(github.com/fortio)` would be better than localmodule

Reply:
the vanity url is fortio.org not github.com/fortio btw

Reply:
Please note that fixing all your issues would be possible simply by running

```sh
gci write -s standard -s default -s "prefix(fortio.org/)"
```